### ISSUE TITLE
Map View and Pagination interaction is confusing

### ISSUE THREAD

**Issue by sampoorna** [covid-maps/covid-maps#118]
Steps to reproduce
1. Go to page 2 or 3 (any page after page 1) on the homepage
2. Select pin on the map
3. The corresponding card is not highlighted

I presume the corresponding card is on page 1, but when you're on a different page, you still expect to see the most relevant card on top. To add to the confusion, the cards sometimes get re-arranged, so you see a different card on top of that page, but it's still not the selected location.

Suggested fix
- Refresh homepage when user selects pin on the map, so that they are back on page 1 and the relevant card is on the top

**Comment by arjun27** [comment]
Thanks for reporting. I'm not happy with the pagination approach, and we should replace it with a load more approach where the results are not split across multiple pages.

**Comment by sampoorna** [comment]
Agreed, is there a ticket for that? We should prioritise that if so.

**Comment by arjun27** [comment]
Updating this one to call it out

**Comment by sampoorna** [comment]
Great, thanks!
